fe/fe-core/src/main/java/org/apache/doris/catalog/ResourceMgr.java

```diff
@@ -17,9 +17,11 @@
 
 package org.apache.doris.catalog;
 
+import com.google.common.collect.Lists;
 import org.apache.doris.analysis.CreateResourceStmt;
 import org.apache.doris.analysis.DropResourceStmt;
 import org.apache.doris.catalog.Resource.ResourceType;
```

Comment:
the resouce has two definations:
1、org.apache.doris.catalog.Resource.ResourceType
2、org.apache.doris.catalog.ResourceType

should be fix

Reply:
`org.apache.doris.catalog.ResourceType` is deprecated. It will be deleted in the future